app/src/org/commcare/android/util/FormUploadUtil.java

```diff
@@ -293,7 +293,7 @@ private static boolean buildMultipartEntity(MultipartEntity entity,
                     fb = new EncryptedFileBody(f, FormUploadUtil.getDecryptCipher(key),
                             ContentType.TEXT_XML);
                 } else {
-                    fb = new FileBody(f, "text/xml");
+                    fb = new FileBody(f, ContentType.TEXT_XML, f.getName());
```

Comment:
@ctsims this was the fix for the submission issue - the deprecated format no longer passes HQ's newly strict requirements. Guessing we'll need/want to do the same for the other formats below

cc @benrudolph